jest.common.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   moduleDirectories: ["node_modules", "<rootDir>"],
   preset: "ts-jest",
-  setupFiles: ["jest-canvas-mock", "../../intersectionObserverMock.js"],
+  setupFilesAfterEnv: ["./jest.setup.ts"],
```

Comment:
Removed the unused jest-canvas-mock dep.

Also moved the intersectionObserver mock and the testing-library/jest-dom setup into a setup file (which makes the typescript integration easier).

Reply:
Oh great idea thanks for doing this!